src/OpenSSL-standalone.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="OpenSSL-standalone" name="OpenSSL License - standalone">
+      <crossRefs>
+         <crossRef>http://www.openssl.org/source/license.html</crossRef>
+      </crossRefs>
+      <notes>
+         This is the first license in the OpenSSL two-license stack as a standalone option.
+      </notes>
+      <text>
+         <titleText>
+            <p>
+               OpenSSL License (standalone)
```

Comment:
Do we see the license title appearing in the wild with "(standalone)" in it? If not, I would probably remove that. The notes and name/identifier help to identify it as standalone.